coins/src/adapters/yield/ember.ts

```diff
@@ -0,0 +1,34 @@
+import { Write } from "../utils/dbInterfaces";
+import { addToDBWritesList } from "../utils/database";
+import axios from "axios";
+import { getTokenInfo } from "../utils/sui";
+import { runInPromisePool } from "@defillama/sdk/build/util";
+
+export async function ember(timestamp: number) {
+  if (timestamp != 0) throw new Error("Ember is only for current price");
+  const res = await axios.get(
+    "https://vaults.api.sui-prod.bluefin.io/api/v1/vaults/coins/price"
+  );
+  const writes: Write[] = [];
+
+  await runInPromisePool({
+    items: res.data,
+    concurrency: 5,
+    processor: async (m: any) => {
+      const { decimals, symbol } = await getTokenInfo(m.coinType);
```

Comment:
would be nice if we cached it, or check against metadata logs pulled from ES, if it exists there, use that symbol and decimals, else pull it here